install.ps1

```diff
@@ -520,25 +520,31 @@ if (-not $noGui.IsPresent) {
              $category = $_.SelectSingleNode("m:properties/d:Tags", $ns).InnerText
              # Select only packages that have the latest version, contain a category and the category is not excluded
              if (($isLatestVersion -eq "true") -and ($category -ne "") -and ($excludedCategories -notcontains $category)) {
-                  $packageName = $_.properties.Id
-                  $description = $_.properties.Description
-
-                  # Initialize category as an empty array
-                  if (-not ($packagesByCategory.ContainsKey($category))) {
-                     $packagesByCategory[$category] = @()
-                  }
-		  # Add the PackageName and PackageDesccription to each entry in the array
-                  $packagesByCategory[$category] += [PSCustomObject]@{
-                     PackageName = $packageName
-                     PackageDescription = $description
-                  }
-               }
-          }
+	             $packageName = $_.properties.Id
+				 $description = $_.properties.Description
+				 $projectUrl = $_.properties.projectUrl
+
+				 # Initialize category as an empty array
+				 if (-not ($packagesByCategory.ContainsKey($category))) {
+					 $packagesByCategory[$category] = @()
+				 }
+				 $packageObject = [PSCustomObject]@{
+				 PackageName        = $packageName
+				 PackageDescription = $description
+				 }
+				 # Check if $projectUrl contains a valid URL
+				 if ($projectUrl -match "^http") {
+					 Add-Member -InputObject $packageObject -MemberType NoteProperty -Name "PackageUrl" -Value $projectURl
```

Comment:
Why not just the code I proposed (simpler and more readable)?
```suggestion
					 $packageObject.PackageUrl = $projectUrl
```

Reply:
This is because I am using the `PsCustomObject`, see my comment https://github.com/mandiant/flare-vm/pull/714#discussion_r2262522105
That doesn't work when using a `PsCustomObject`, see error : `ForEach-Object : Exception setting "PackageUrl": "The property 'PackageUrl' cannot be found on this object. Verify that the property exists and can be set."`

Reply:
As I said above, do not use `PsCustomObject` (instead of a simple hash table) if there is no reason to (I do not see a reason here), specially if it complicates the code, as it happens here. The code with a `PsCustomObject` is much more difficult to read than with a simple hash table.